website/docs/r/servicehooks_permissions.html.markdown

```diff
@@ -0,0 +1,72 @@
+---
+layout: "azuredevops"
+page_title: "AzureDevops: azuredevops_servicehooks_permissions"
+description: |-
+  Manages permissions for AzureDevOps service hooks
+---
+
+# azuredevops_servicehooks_permissions
+
+Manages permissions for service hooks
+
+## Permission levels
+
+Permissions for service hooks within Azure DevOps can be applied only on Project level.
```

Comment:
If my understanding is right, the permission can be assigned to organization level if `project_id` not set. ping @tmeckel   if I'm wrong

Reply:
@xuzhang3 excatly your're right! Thus the `project_id` is optional. @martenvd you can copy (or compare) my documentation